accepted/rfc-process.md

```diff
@@ -0,0 +1,109 @@
+# RFC process for the Bytecode Alliance
+
+# Summary
+
+As the Bytecode Alliance grows, we need more formalized ways of communicating about and reaching consensus on major changes to core projects. This document proposes to adapt ideas from Rust’s [RFC](https://github.com/rust-lang/rfcs/) and [MCP](https://forge.rust-lang.org/compiler/mcp.html) processes to the Bytecode Alliance context.
```

Comment:
There's a formal proposal for the MCP process at https://github.com/rust-lang/rfcs/pull/2936/ ; I would suggest linking to that.

Also, please expand "RFC" and "MCP" on first use.

Reply:
I'll expand the acronyms :+1:

Re: the MCP link, I'd prefer to keep it pointed at the official documentation on the Forge, which post-dates the compiler team RFC (which was accepted a while back).